experimenter/experimenter/experiments/models.py

```diff
@@ -1347,6 +1353,124 @@ def get_branch_data(self, analysis_basis, selected_segment):
 
         return branch_data
 
+    def formattedAnalysisPointComparator(self, point):
+        wi = point.get("window_index")
+        return int(wi) if wi is not None else 0
+
+    def get_metric_area_data(
```

Comment:
Not something to do in this PR but it might make it cleaner/easier to read if we move all these methods up above the class into a little mixin of just the methods/properties for the results, so they're all in one place.  Can do as a followup later.